cmd/wallet-web/src/pages/chapi/Signup.vue

```diff
@@ -1,42 +1,50 @@
 <template>
-  <div class="gradient">
+  <!-- Todo  Move this style to theme setup -->
+  <div class="bg-gray-light min-w-screen min-h-screen flex items-center justify-center px-5 py-5 bg-scroll bg-onboarding bg-no-repeat">
     <!-- This is an example component -->
-    <div class="px-4 py-24">
-      <div
-          class="md:max-w-6xl md:mx-auto md:flex md:items-center md:justify-between">
-        <div class="flex justify-between items-center">
-          <img class="h-8 w-8 mr-2" src="@/assets/img/logo.png" alt="">
-          <a class="font-semibold text-white text-2xl lg:text-4xl tracking-tight" href="javascript:history.back()">TrustBloc</a>
-          <div
-              class="inline-block cursor-pointer md:hidden">
+    <div class="bg-gradient-dark rounded-xl w-full overflow-hidden text-neutrals-softWhite" style="max-width:896px; max-height:467px;">
```

Comment:
As discussed, Anton have a margins defined in the themes in his pr. Once he merged  i will update this with defined margin rather than spending time to figure out accurate spacing.